src/main/java/com/jcabi/github/RtBranches.java

```diff
@@ -103,6 +104,8 @@ public Branch map(final JsonObject object) {
 
     @Override
     public Branch find(final String name) {
-        throw new UnsupportedOperationException("find(name) not supported");
+        return StreamSupport.stream(this.iterate().spliterator(), false).filter(
+            item -> item.name().equals(name)
+        ).findFirst().orElse(null);
```

Comment:
@paulodamaso Should we really return ``null`` if we don't find it? What does the Github API say, maybe we can create it if we don't find an existing one?

Reply:
@amihaiemil Unfortunately all we have on Github API is https://developer.github.com/v3/repos/branches/#get-branch , and it doesn't say nothing about a seearch without result. I did not liked this `null` either, but I don't know if it is right to create a new branch right here on this method; maybe we could implement something like a `findOrCreate´ method later, WDYT?